lib/generators/templates/landable.rb

```diff
@@ -51,6 +51,31 @@
 
   # If you want to save a different UserAgent if the request.user_agent is blank, set it here
   # config.blank_user_agent_string = 'blank'
+
```

Comment:
there are still a couple of references to Hutch in these comments, we can get rid of those now, and update them to Bunny